@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -105,17 +105,13 @@ describe("MapView", function() {
             removeEventListener: removeEventListenerSpy
         } as unknown) as HTMLCanvasElement;
         mapViewOptions = {
-            canvas,
-            // Both options cause the `addDataSource` method to be called, which we can't `await` on
-            // because it is called in the constructor, but we can disable them being added.
-            addBackgroundDatasource: false,
-            enablePolarDataSource: false
+            canvas
         };
     });
 
     afterEach(async function() {
         if (mapView !== undefined) {
-            await mapView.getTheme();
+            await mapView.loadPromise();
```

Comment:
same here, I think dispose should work without any API call to be waited for, or dispose should do so internally if needed